types/chrome/index.d.ts

```diff
@@ -1576,7 +1589,7 @@ declare namespace chrome.contextMenus {
          */
         onclick?: ((info: OnClickData, tab: chrome.tabs.Tab) => void) | undefined;
         /** Optional. The ID of a parent menu item; this makes the item a child of a previously added item.  */
-        parentId?: any;
+        parentId?: number | string;
```

Comment:
https://developer.chrome.com/docs/extensions/reference/contextMenus/#method-create

Reply:
Since the `exactOptionalPropertyTypes` typescript flag is set, undefined should be explicitly declared.
```suggestion
        parentId?: number | string | undefined;
```